Makefile.doc

```diff
@@ -65,7 +65,7 @@ REFMANCOQTEXFILES:=$(addprefix doc/refman/, \
 
 REFMANTEXFILES:=$(addprefix doc/refman/, \
   headers.sty Reference-Manual.tex \
-  AsyncProofs.tex) \
+  RefMan-uti.tex) \
```

Comment:
There is still an issue here (must have happened during the rebase).
This file shouldn't have changed in this commit.

Reply:
@Zimmi48 revert the change of the makefile

Reply:
What do you mean exactly?